client/src/components/sideBar/SideBar.js

```diff
@@ -36,31 +45,63 @@ export function SideBar() {
 
   const tripEntries = useMemo(() => {
     return trips.map((trip) => {
+      const buttonColor =
+        trip.id === activeTripId &&
+        (appView === AppView.TRIP_VIEW || appView === AppView.DAY_VIEW)
+          ? 'bg-green-300/40 font-medium hover:bg-green-400/40'
+          : 'bg-slate-300/40 hover:bg-slate-400/40'
+
       return (
         <Button
           key={`sidebar-trip-${trip.id}`}
           onClick={() => {
             dispatch(setActiveTripId(trip.id))
           }}
-          className='w-full bg-opacity-40 hover:bg-opacity-40'
-          active={
-            trip.id === activeTripId &&
-            (appView === AppView.TRIP_OVERVIEW || appView === AppView.TRIP_DAY)
-          }
+          className={`w-full ${buttonColor}`}
         >
           {trip.tripName}
         </Button>
       )
     })
   }, [trips, activeTripId, appView])
 
+  const preferencesModalBtn = useMemo(() => {
+    // TODO refactor this to return a button that opens a modal.
+    //    Please follow any conventions that Meng uses when styling / storing modal content
+    return (
+      <div className='mb-2 w-full rounded-md bg-slate-100/40 p-2'>
+        <h3 className='mb-1 text-lg font-semibold'>Preferences</h3>
+        <Toggle
+          className='mb-2'
+          label='Compact View'
+          onClick={() => {
+            dispatch(toggleCompactView())
+          }}
+          active={isCompactView}
+        />
+        <Toggle
+          label='Vertical Timelines'
+          onClick={() => {
+            dispatch(toggleVerticalTimelines())
+          }}
+          active={isVerticalTimelines}
+        />
+      </div>
+    )
+  }, [isVerticalTimelines, isCompactView])
+
```

Comment:
also for this one, do you mean (in the future) that we will need to place the toggles inside the modal?

Reply:
nvm i forgot to read your comment lmao